doc/Manuals/LoadBalancing/ig-lb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -0,0 +1,691 @@
+* Changes to the `serverconf` database, authorization and signing keys are applied via the configuration master, which is
+  a member of the cluster. The replication is one-way from master to slaves and the slaves should treat the configuration
+  as read-only.
+* The cluster nodes can continue operation if the master fails but the configuration can not be changed until:
+  - the master becomes back online, or
+  - some other node is manually promoted to be the master.
+* If a node fails, the messages being processed by that node are lost.
+  - It is the responsibility of the load balancer component to detect the failure and route further messages to other nodes.
+    Because there potentially is some delay before the failure is noticed, some messages might be lost due to the delay.
+  - Recovering lost messages is not supported by the load balancing support.
+* Configuration updates are asynchronous and the cluster state is eventually consistent.
+* If the master node fails or communication is interrupted during a configuration update, each slave should have a valid
+  configuration, but the cluster state can be inconsistent (some members might have the old configuration while some might
+  have received all the changes).
+  
+### 2.2 Communication with external servers and services: The cluster from the point of view of a client or service
+
+When external security servers communicate with the cluster, they see only the public IP address of the cluster which is
+registered to the global configuration as the security server address. From the caller point of view, this case is analogous
+to making a request to a single security server.
+
+![alt-text](load_balancing_traffic.png)
+
+When a security server makes a request to an external server (security server, OCSP, TSA or a central server), the
+external server sees only the public IP address. Note that depending on the configuration, the public IP address
+might be different from the one used in the previous scenario. It should also be noted that the security servers will
+independently make requests to OCSP and TSA services as well as the and central server to fetch the global configuration
+as needed.
+
+![alt-text](load_balancing_traffic-2.png)
+
+### 2.3 State replication from the master to the slaves
+
+![alt-text](load_balancing_state_replication.png)
+
+
+#### 2.3.1 `serverconf` database replication
+| Data            | Replication          | Replication method                                 |
+| ------------------- | -------------------- | -------------------------------------------------- |
+| serverconf database | **replication required** | PostgreSQL streaming replication (Hot standby) |
+
+The serverconf database replication is done using streaming replication with hot standby. Note that PostgreSQL replication
+is all-or-nothing, it is not possible exclude databases from the replication. This is why the replicated serverconf and
```

Comment:
> "not possible exclude"

 -> "not possible to exclude". Also the preceding "," should be ":" as the final clause is a direct explanation.